docs/05-PipelineLaunchOptions.md

```diff
@@ -0,0 +1,99 @@
+# 5.0 - Pipeline Launch options
+
+Scans upload into LORIS and insertion pipeline's triggering can be done in a few 
+ways depending on each project's needs. The best choice will depend on whether 
+for example data is collected retrospectively or prospectively, and/or whether 
+the MRI protocol is harmonized across all the sites involved in the study.
+
+The different options available are graphically illustrated in the graph below.
+
+![PipelineLaunchOptions](images/PipelineLaunchOptions.png)
+
+Regardless of how the project chooses to upload scans and trigger the insertion, 
+the automated quality control and MRI protocol checks performed by the pipeline 
+should remain identical.
+
+It is also worth mentioning that all the options illustrated here assume that 
+the candidate and visit are already registered in the LORIS database.
+
+
+In the next three sub-sections, the three launch options illustrated in the 
+figure above will be briefly highlighted and the exact command needed to launch 
+the pipeline shown.
+Details about the scripts themselves can be found in the [Scripts](04-Scripts.md)
+section.
+ 
+## 5.1 Option 1
+
+Triggering the pipeline is done from the `/data/$PROJECT/bin/mri` directory as 
+follows:
+
+```
+uploadNeuroDB/imaging_upload_file.pl -profile prod -upload_id $UploadIDID /PATH/TO/UPLOAD/PSCID_CandID_VisitLabel_OptionalSuffix.zip -verbose
```

Comment:
I don't think you require "uploadNeuroDB/" when calling the script since it is supposed to be in the $PATH variable. Technically, you could trigger imaging_upload_file.pl from anywhere, no?

Reply:
you could, BUT, 1) our scripts internally are called/run this way (they all call the bin/mri/ directory and append the uploadNeuroDB/ to the path), and 2) users that want to do auto-complete (like I do), have to put the leading path...